### PR TITLE
feat: Added a express/knex example application

### PR DESCRIPTION
**Pull request by bizob2828**
This is the example application I built to demonstrate behavior of express/knex with hybrid agent. I was intentional sparse on the README as we're still doing research on the hybrid agent but wanted this in our repo for other engineers to use

### CODE REVIEW

File: knex/config/database.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+'use strict'
+
+const env = process.env.NODE_ENV || 'development'
```
Comment (jsumners-nr) [inline knex/config/database.js:3]: Do we need `NODE_ENV` things?

File: knex/config/database.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+const knexfile = require('../knexfile')
+debugger
+const knex = require('knex')(knexfile[env])
```
Comment (jsumners-nr) [inline knex/config/database.js:6]:
```suggestion
const knexfile = require('../knexfile')
const knex = require('knex')(knexfile[env])
```

File: knex/LICENSE
[the remark this answers is not in the document]
Comment (bizob2828) [reply knex/LICENSE:1]: so this was an existing app out there so thought i'd keep this for attribution but open to other ways to do that

File: knex/config/database.js (hunk 1 above)
Comment (bizob2828) [reply knex/config/database.js:3]: again, this was an example online. i can pare down but not sure if it really matters

File: knex/LICENSE
[the remark this answers is not in the document]
Comment (jsumners-nr) [reply knex/LICENSE:1]: I did not realize that.

**Review by jsumners-nr — APPROVED** [review]

File: knex/LICENSE
[the remark this answers is not in the document]
Comment (bizob2828) [reply knex/LICENSE:1]: yea sorry i should've mentioned that up front

**Pull request merged**